lib/SVF-FE/LLVMModule.cpp

```diff
@@ -227,18 +228,79 @@ void LLVMModuleSet::initialize()
     }
 }
 
+std::vector<llvm::Function *> LLVMModuleSet::getLLVMGlobalFunctions(
+        llvm::GlobalVariable *global)
+{
+    class LLVMGlobalFunction {
+    // llvm.global_ctors and llvm.global_dtors
+    public:
+        unsigned priority;
+        llvm::Function *func;
+        LLVMGlobalFunction(){};
+        LLVMGlobalFunction(unsigned _priority, llvm::Function *_func)
+              : priority(_priority), func(_func){};
+        bool operator>(const LLVMGlobalFunction &other) const
+        {
+            return priority > other.priority;
```

Comment:
Do we change to priority > other.priority && _func > other._func if two globalfunctions' priority is the same?